docs_user/modules/proc_adopting-key-manager-service.adoc

```diff
@@ -38,7 +38,6 @@ spec:
       serviceAccount: barbican
       serviceUser: barbican
       passwordSelectors:
-        database: BarbicanDatabasePassword
```

Comment:
Hackfest comment for reference: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABQ54dA3g